.github/workflows/target-determination-indexer.yml

```diff
@@ -0,0 +1,114 @@
+name: Index PyTorch Tests for Target Determination
+
+on:
+  workflow_dispatch:
+  # TODO: Trigger every few hours
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  index:
+    runs-on: linux.g5.4xlarge.nvidia.gpu # 1 GPU A10G 24GB each
```

Comment:
I think you will need to call setup-linux https://github.com/pytorch/pytorch/blob/main/.github/workflows/_linux-test.yml#L71-L72 and install NVIDIA driver https://github.com/pytorch/pytorch/blob/main/.github/workflows/_linux-test.yml#L94-L97 here.  If this job is run on a newly launch g5 runner by chance, it would not a CUDA driver IIRC, so the indexing step would fail

Reply:
Hmm that's strange, looks like indexing passed in this job on GPU:  https://github.com/pytorch/pytorch/actions/runs/7849903772/job/21424841173?pr=118824. Will investigate in a follow-up.

Reply:
Oh, the runner is reusable, as long as the indexing job is not the first one to run there, the driver will be there.  However, my understanding might be outdated, may be we already have included CUDA driver in the image (although I doubt that)